src/sql/parts/jobManagement/views/jobHistory.component.ts

```diff
@@ -284,22 +257,45 @@ export class JobHistoryComponent extends Disposable implements OnInit {
 	}
 
 	private setActions(): void {
-		let startIcon: HTMLElement = $('.icon-start').get(0);
-		let stopIcon: HTMLElement = $('.icon-stop').get(0);
+		let startIcon: HTMLElement = $('.runJob').get(0);
```

Comment:
is there a way to make these ids a little less generic?  runJob and stopJob seem like they could collide at some point in the future.